gateway/api/server.go

```diff
@@ -148,6 +150,15 @@ func (api *Api) buildRoutes(route *gin.RouterGroup) {
 	route.GET("/login", loginHandler.Login)
 	route.GET("/callback", loginHandler.LoginCallback)
 	route.GET("/healthz", apihealthz.LivenessHandler())
+	///////////////////////
+	// local auth routes //
+	///////////////////////
+	// TODO do not allow these routes be called if auth is not set as local
```

Comment:
Was this TODO made? Sounds important

Reply:
good catch

Reply:
as we plan to expand local auth, I've created a middleware called `LocalAuthOnly` to manage this